src/core/sys/posix/stdio.d

```diff
@@ -344,3 +344,7 @@ unittest
     assert(memcmp(ptr, testdata.ptr, testdata.length*wchar_t.sizeof) == 0);
     assert(fclose(f) == 0);
 }
+
+
+ssize_t getdelim (char** lineptr, size_t* n, int delimiter, FILE* stream);
+ssize_t getline (char** lineptr, size_t* n, FILE* stream);
```

Comment:
I just saw the [forum post](http://forum.dlang.org/thread/iioxwplvezvfsrjfdtwh@forum.dlang.org) (thanks for bringing this to Walter's attention) and I was wondering if we should change the declarations to:

```D
ssize_t getdelim(ref scope char* lineptr, scope size_t* n, int delimiter, FILE* stream);
ssize_t getline(ref scope char* lineptr, scope size_t* n, FILE* stream);
```

relying on the fact that the calling convention would remain the same. @WalterBright  do you think this is a good idea?